addons/product/models/product_attribute_value.py

```diff
@@ -126,11 +134,22 @@ def unlink(self):
     def _without_no_variant_attributes(self):
         return self.filtered(lambda pav: pav.attribute_id.create_variant != 'no_variant')
 
-    def action_open_product_template_attribute_value(self):
+    def action_open_attribute_value_wizard(self):
+        self.ensure_one()
+        is_new_attribute_value = not bool(self.pav_attribute_line_ids)
```

Comment:
Nit: the `not` will ensure that the result is a boolean, so I don't think the `bool` cast is needed here.